compose.yaml

```diff
@@ -6,23 +6,13 @@ services:
       - database
     restart: unless-stopped
     ports:
-      # HTTP
-      - target: 8080
-        published: ${HTTP_PORT:-8080}
-        protocol: tcp
-      # HTTPS
-      - target: 443
-        published: ${HTTPS_PORT:-443}
-        protocol: tcp
-      # HTTP/3
-      - target: 443
-        published: ${HTTP3_PORT:-443}
-        protocol: udp
+      - 8080:8080
```

Comment:
Revert this change, it is totaly OK to have it since the API could be exposed to outside.
If you want to modify default port forwarding just use environment variables on your local configuration like so:

```
HTTP_PORT=54321 docker-compose up
```
(or any other values of HTTP/HTTPS/HTTP3 ports you want)


It would also be nice to have it as an environment variable for the `web` container.

Reply:
The API doesn't listen to 443 and 80 so the declaration of these ports are useless  (https://github.com/DevGirl-Team/tamagokill/blob/665d9616757c4bce55fe4bbbd7237de688234830/api/src/main.rs#L22) 

Ok for the local port configuration as env 👍 Will change the code

Reply:
Agreed on the TLS/HTTP3 ports **but** I think it's not that problematic to keep them for now, since it makes sense for the future.